integration/integration_test.go

```diff
@@ -159,9 +159,9 @@ func testMain(m *testing.M) int {
 		if *k8sDistro == K8sDistroARO {
 			livenessProbe = false
 		}
-		if *k8sDistro == K8sDistroMinikubeGH {
-			imagePullPolicy = "Never"
-		}
+// 		if *k8sDistro == K8sDistroMinikubeGH {
+// 			imagePullPolicy = "Never"
+// 		}
```

Comment:
Out of curiosity, why do you need to do this to test AKS?

Reply:
I wonder if that is related to the tag fix we did [here](https://github.com/kinvolk/inspektor-gadget/pull/955/files#diff-bda633e49a0d242b6842493c3896d4f766f361166fa4c08e1c2e01d827544cadR211). If the run failed on loading the image, reverting [image_tag](https://github.com/kinvolk/inspektor-gadget/pull/897/files#diff-bda633e49a0d242b6842493c3896d4f766f361166fa4c08e1c2e01d827544cadL546) and the changes here will fix it.

Reply:
Makes sense RTFM for me :laughing: Wanted to make sure I didn't break anything :laughing: